functional/tpm_policy-sanity-on-localhost/test.sh

```diff
@@ -0,0 +1,95 @@
+#!/bin/bash
+# vim: dict+=/usr/share/beakerlib/dictionary.vim cpt=.,w,b,u,t,i,k
+. /usr/share/beakerlib/beakerlib.sh || exit 1
+
+
+rlJournalStart
+
+    rlPhaseStartSetup "Do the keylime setup"
+        rlRun 'rlImport "./test-helpers"' || rlDie "cannot import keylime-tests/test-helpers library"
+        limeBackupConfig
+        # if IBM TPM emulator is present
```

Comment:
This comment is outdated, no? I think now we check for `swtpm`?

Reply:
Right. I will remove it.. I guess there are more files with this comment.